docs/alerts/creating-heartbeat-alerts.md

```diff
@@ -23,13 +23,13 @@ Using these tags you can do things like:
 - Grouping by `container.id` to get alerted when that particular container dies. Yes, this could lead to a lot of alert noise, so use this sparingly and wisely.
 - Grouping by `os.host` and `container.image.name` to get alerted when a host stops running all containers with some image name. The container may have been moved to another host.
 
-This example will show how to include two specific images we’re interested in and grouping them by `os.host` and `container.image.name` to get alerted when a host stops running all containers with a particular image name.
+The following example shows how to include two specific container images we are interested in.  We group them by `os.host` and `container.image.name` to get alerted when any one host stops running all containers with a particular image name.
```

Comment:
Here I am not sure.
We `include two specific images we’re interested` by filtering and then group by to be able separate/distinguish them between each other.
Your version sounds for me like `to include 2 container images we group by them`

Reply:
Simplified, I hope...